internal/devbox/packages.go

```diff
@@ -516,7 +529,6 @@ func (d *Devbox) installNixPackagesToStore(ctx context.Context, mode installMode
 		args.AllowInsecure = allowInsecure
 		err = nix.Build(ctx, args, installables...)
 		if err != nil {
-			color.New(color.FgRed).Fprintf(d.stderr, "Fail\n")
```

Comment:
is deleting this a good idea for non-nixcache scenarios (i.e. the common scenario)?

Reply:
Yeah, we print error a few times (once for nix stderr, and once for devbox failing) so this was a third error (and not very informative because it was just `fail`). The nix output tends to be the most informative.